UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -293,4 +317,13 @@ private IEnumerator<WaitForSeconds> CooldownTick()
         yield return new WaitForSeconds(internalBleedingCooldown);
         onCooldown = false;
     }
+
+    [Serializable]
+    class ToxicGases
```

Comment:
```suggestion
    class ToxicGas
```
Each of these represent a single entry